designdocs/capacity.md

```diff
@@ -23,89 +23,105 @@ Frequency explains how Capacity can be acquired through staking, refills, and us
 - [Block space allocation for Capacity transactions](#block-space)
 - [Implementation of spending Capacity to perform transactions](#capacity-transactions)
 
-  **Implementation of how to acquire through staking:** <a id='staking'></a>
+### **Implementation of how to acquire Capacity through staking:** <a id='staking'></a>
```

Comment:
markdown lint complains if there is no heading, so I tried to make the headings make sense.